front/src/styles/scss/applications/stdcmV2/_home.scss

```diff
@@ -169,16 +176,26 @@
         }
       }
     }
-    .simuation-results {
+    .simulation-results {
       display: flex;
       justify-content: space-between;
+      .map-results {
+        border: 0.063rem solid rgba(255, 255, 255, 1);
+        border-radius: 0.5rem;
+        box-shadow:
+          0rem 0rem 0rem 0.125rem rgba(255, 255, 255, 0.75) inset,
+          0rem 0rem 0rem 0.063rem rgba(0, 0, 0, 0.25) inset;
+        height: 33.75rem;
+        margin: 2rem 2rem 0 0.85rem;
+        width: 100%;
```

Comment:
Would probably be a tad better to use `flex-grow: 1` instead of `width: 100%` since the parent is a flex container.

Reply:
TBH it's a bit strange to me to make the map width adaptive, I would've expected the table on the left side to be `flex-grow: 1` and the map to have a fixed size?